handler/incoming_instance_handler.go

```diff
@@ -237,6 +245,8 @@ func (hdlr *InstanceEventHandler) setOrderAndNodeIDs(ctx context.Context, instan
 		}
 	}
 
```

Comment:
I think the time needs sampling here as:
t1 := time.Since(t0)
and then use it in the log line, to avoid including any timing of any code in the log line